packages/uswds-utilities/src/styles/palettes/_index.scss

```diff
@@ -1,5 +1,5 @@
-@forward "color-palletes";
-@forward "default-palletes";
-@forward "font-palletes";
+@forward "color-palettes";
+@forward "default-palettes";
+@forward "font-palettes";
 @forward "palette-registry";
-@forward "spacing-palletes";
+@forward "spacing-palettes";
```

Comment:
This index file appears to be here only as a formality and for consistency with other directories of Sass files. It is not referenced in the codebase. I was concerned that this might be a breaking change, but it appears not